src/games/strategy/sound/ClipPlayer.java

```diff
@@ -271,17 +271,14 @@ private void playClip(final String clipName, final PlayerID playerId) {
 
     final URI clip = loadClip(folder);
     if (clip != null) {
-      (new Thread(() -> {
-        try {
-          Optional<InputStream> inputStream = UrlStreams.openStream(clip.toURL());
-          if(inputStream.isPresent()) {
-            final AudioDevice audioDevice = FactoryRegistry.systemRegistry().createAudioDevice();
-            new AdvancedPlayer(inputStream.get(), audioDevice).play();
-          }
-        } catch (Exception e) {
-          ClientLogger.logError("Failed to play: " + clip, e);
-        }
-      })).start();
+      SwingUtilities.invokeLater(() -> {
+        new JFXPanel();// This loads the JavaFX Toolkit
```

Comment:
Would be nice if we could create the JFXPanel once, and not every time we play a sound. Not a big deal though, objects are generally quite cheap in java8.